packages/graphql/src/graphql/ops/MUpdateQuestionInstances.graphql

```diff
@@ -0,0 +1,5 @@
+mutation UpdateQuestionInstances($questionId: Int!) {
+  updateQuestionInstances(questionId: $questionId) {
+    id
```

Comment:
@rschlaefli Should we return the entire instances here to update the cache properly? I left it out for now as the instances should hopefully be fetched together with the corresponding element anyways?